client/src/components/mapping/TabColumnPicking/StaticValueForm/index.tsx

```diff
@@ -138,14 +178,96 @@ const StaticValueForm = ({ attribute }: Props) => {
       await createStaticInput({
         variables: {
           attributeId,
-          staticValue
+          staticValue: customSystem
```

Comment:
It bothers me that we have to put this logic directly inside the component but I can't think of anywhere else to put it

Reply:
Agreed, but it's really the component that does the job...